.editorconfig

```diff
@@ -0,0 +1,19 @@
+# http://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+max_line_length = 80
+trim_trailing_whitespace = true
+quote_type = single
+
+[*.md]
+max_line_length = 0
+trim_trailing_whitespace = false
+
+[COMMIT_EDITMSG]
+max_line_length = 0
```

Comment:
Minor thing, could you update your eslint or prettier to automatically add a blank line at the end of each file to fix this github warning.

Reply:
I would say to leave that to the issue #21